### ISSUE TITLE
Why the size of the detected pictures have been changed?

### ISSUE THREAD

**Issue by hnuzhy** [MVIG-SJTU/AlphaPose#57]
When I try to use the results of the detection "alpha-pose-results-forvis.json" to draw the keypoints on the origin pictures, I find that the position of all the keypoints are displaced compared with the detected pictures in folder RENDER. 
Then I checked the size of the origin pictures and the detected pictures, the size of the latter has been resized larger. However, when I resized the origin pictures to the width x height which is the same as detected pictures, question still keep there. I also tried to change the coordinates of the keypoints in the json file, as a result, these points are still in the wrong position. 
So I'm wondering if there are any problems with these coordinates of all keypoints?

**Comment by Fang-Haoshu** [comment]
We haven't met such problem before. Can you share some results?

**Comment by hnuzhy** [comment]
Ok, here is a example
the origin picture with size 1920x1080
![image](https://user-images.githubusercontent.com/21312423/39671370-92d58bbe-5149-11e8-8635-79ed0b8ff55e.png)

the detected picture with size 1934x1123
![image](https://user-images.githubusercontent.com/21312423/39671360-541eb79c-5149-11e8-94b7-27a0ddbb27ec.png)

when I try to mark the key points in the origin picture using the data in "alpha-pose-results-forvis.json", 
the positons of them seem different from the detected picture
![image](https://user-images.githubusercontent.com/21312423/39671389-e85aeb42-5149-11e8-8e76-275c28a87f44.png)

**Comment by hnuzhy** [comment]
No matter whether I adjust the size of the origin picture or change the coordinates of all key points in proportion, the problem just can not be solved perfectly.

**Comment by Fang-Haoshu** [comment]
Oh I see, the picture in the RENDER is 1934x1123 because there will be some padding to the original image when saving the file.

The keypoints in alpha-pose-results-forvis.json are in the corrent location in the size of 1920x1080.

You don't need to adjust the size of the origin picture or change the coordinates of all key points in proportion

**Comment by hnuzhy** [comment]
Thanks a lot for replying!
But as you can see in the third picture, the places of the key points are still biased. So how could I solve this problem？
Could you please tell me the detailed padding strategy?

**Comment by Fang-Haoshu** [comment]
Do you input the image with the red box?
I don't think the keypoints you use in the third image is the same as the second image, because apparently, the angles are different

**Comment by hnuzhy** [comment]
Sorry for giving the origin picture with the red box. Actually, I didn't draw the red box in the input images.
And I truly used the positions of all key points given in file "alpha-pose-results-forvis.json", so I don't know where the problem is......

**Comment by Fang-Haoshu** [comment]
That's weird... Check your program of drawing keypoints maybe.
We can see that the line of the head has different angles in two images, which can not be introduced by size changes.

**Comment by Fang-Haoshu** [comment]
You are welcome. If you find any possible reasons, please tell me about that. I am also very interested.

**Comment by hnuzhy** [comment]
Thank you for your help yesterday, I found where the problem is finally!
when I load the json file "alpha-pose-results-forvis.json" in my python script, I get the Dict "rmpe_results". The keys of it are names of original images. However, these names are unsorted, and I didn't find this at the beginning. So I just applied _`list(rmpe_results.keys()).sort()`_, the problem was solved.  It's so careless of me :(
